src/utils/googleAnalytics.ts

```diff
@@ -14,27 +15,49 @@ export const COOKIES_LIST = [
   { name: '_gid', path: '/' },
 ]
 
+const IS_STAGING = getCurrentEnvironment() === 'staging'
+const shouldUseGoogleAnalytics = IS_PRODUCTION || IS_STAGING
+
+export const trackGAEvent: typeof ReactGA.event = (...args) => {
+  return shouldUseGoogleAnalytics ? ReactGA.event(...args) : console.info('[GA] - Event:', ...args)
+}
+const trackGAPageview: typeof ReactGA.pageview = (...args) => {
+  return shouldUseGoogleAnalytics ? ReactGA.pageview(...args) : console.info('[GA] - Pageview:', ...args)
+}
+
 let analyticsLoaded = false
 export const loadGoogleAnalytics = (): void => {
   if (analyticsLoaded) {
     return
   }
-  // eslint-disable-next-line no-console
-  console.log('Loading google analytics...')
-  const trackingID = getGoogleAnalyticsTrackingID()
+
+  console.info(
+    shouldUseGoogleAnalytics
+      ? 'Loading Google Analytics...'
+      : 'Google Analytics will not load in the development environment, but log instead.',
+  )
+
+  const gaTrackingId = getGoogleAnalyticsTrackingID()
   const networkInfo = getNetworkInfo()
-  if (!trackingID) {
-    console.error('[GoogleAnalytics] - In order to use google analytics you need to add an trackingID')
+  const customDimensions = {
+    anonymizeIp: true,
+    appName: `Gnosis Safe Multisig (${networkInfo.label})`,
+    appId: `io.gnosis.safe.${networkInfo.label.toLowerCase()}`,
+    appVersion: process.env.REACT_APP_APP_VERSION,
+  }
+
+  if (shouldUseGoogleAnalytics) {
+    if (!gaTrackingId) {
+      console.error('[Google Analytics] - In order to use Google Analytics you need to add a tracking ID.')
```

Comment:
Here we are setting [Google Analytics] instead of [GA]
Shouldn't we be consistent here?